bin/common.sh

```diff
@@ -72,7 +72,7 @@ source ${BK_CONFDIR}/bk_cli_env.sh
 
 detect_jdk8() {
 
-  if [ -f "$JAVA_HOME/bin/jshell" ]; then
+  if [ -f "$JAVA_HOME/lib/modules" ]; then
```

Comment:
How does this differentiate between JDK8 and JDK11?

I see this for JDK11
```
 ~> ls $JAVA_HOME/lib/modules                                                                                                                                                                                                                                                                       
/Library/Java/JavaVirtualMachines/openjdk-11.0.2.jdk/Contents/Home/lib/modules
```

Reply:
Note: This is on a mac, is this script designed to be run on all platforms? @eolivelli

Reply:
I have developed this fix on Linux.

We only have to detect that we are on JDK9+ because of lots of command line options that have been dropped and prevent the JVM to start.
The main difference in JDK9 is the JPMS (Module System) so I think this is a good "test".
In JDK8 there is no "lib/modules" file, it contains all of the java "modules".
In the "JDK" and not JRE version there is also a lot of ".jmods" files, but they are not present in the single JRE version.
I don't want to execute "java -version" because we will have to parse the result and it will be too hard to maintain.

I have tested with AdoptOpenJDK and with Zulu JDK (both JDK and JRE)